odoo/addons/base/models/ir_http.py

```diff
@@ -27,8 +27,8 @@
     slugify_lib = None
 
 import odoo
-from odoo import api, http, models, tools, SUPERUSER_ID
-from odoo.exceptions import AccessDenied
+from odoo import _, api, http, models, tools, SUPERUSER_ID
```

Comment:
So... we've got that thing, an unspoken order for the `from odoo import` :grimacing: 
```suggestion
from odoo import api, http, models, tools, _, SUPERUSER_ID
```

Reply:
wuuuuuut, really? damn!